lib/HTFeed/Stage/ImageRemediate.pm

```diff
@@ -258,7 +256,8 @@ sub _remediate_tiff {
 
     # Does it look like a contone? Bail & convert to JPEG2000 if so.
     if(!$bad and $fields->{'IFD0:BitsPerSample'} eq '8'
-            or $fields->{'IFD0:SamplesPerPixel'} eq '3') {
+        and ($fields->{'IFD0:SamplesPerPixel'} eq '3'
+             or $fields->{'IFD0:SamplesPerPixel'} eq '1')) {
```

Comment:
Took me a minute to parse out where the parentheses are, but I think this looks right.

Reply:
yeah we don't have rubocop to fix our terrible formatting. the exiftool fields are definitely ripe for some class extraction as a wrapper around them rather than treating them as a hash, but not today...